### ISSUE TITLE
What is the image token for contacts?  

### ISSUE THREAD

**Issue by Berlira** [mautic/mautic#4231]
What type of report is this:

| Q  | A
| -Y--| ---
| Bug report? | 
| Feature request? | 
| Enhancement? | 

## Description:
What is the image token for the contacts?  I would like to drop it in the emails.  Thanks, V. 

## If a bug:

| Q   | A
| --- | ---
| Mautic version | 
| PHP version | 

### Steps to reproduce:
1. 
2. 
 
### Log errors: 

_Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable._

**Comment by escopecz** [comment]
Do you mean contact's avatar? Or what image?

**Comment by Berlira** [comment]
Yes the contact profile image.  What is the token for this?  Thanks.

**Comment by escopecz** [comment]
I don't think there is one at the moment. I'll mark this as a feature request.

**Comment by Berlira** [comment]
Ok.  Thanks so much for this.
